.github/workflows/on-push-master-publish-chart.yml

```diff
@@ -0,0 +1,40 @@
+name: Publish helm chart
+
+on:
+  push:
+    branches:
+      - master
+      - main # assume that the branch name may change in future
+    paths:
+      - 'charts/**' 
+      - '.github/**'     
+env:
+  KUBE_SCORE_VERSION: 1.10.1
+  HELM_VERSION: 3.5.2
+
+jobs:
+  publish-chart:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+
+      - name: Install Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v${{ env.HELM_VERSION }}
+
+      - name: Run chart-releaser
```

Comment:
Will this publish to the root of repo in the gh-pages branch? We've talked recently about using GitHub pages to host other content, like our user manual and other docs, so if we're going to publish the Helm charts there I would rather have those charts in a subdirectory of the gh-pages.